service_common.go

```diff
@@ -141,6 +141,39 @@ func newService(name string, id string, ingressPorts []*PortConfig, serviceAlias
 	}
 }
 
+func (c *controller) getLBIndex(sid, nid string, ingressPorts []*PortConfig) int {
```

Comment:
This function is not used by this commit. I dont think adding this new function to resolve the conflict is appropriate.

Reply:
i've updated the commit a75ba12 to not include `getLBIndex` in `service_common.go`